crm_claim/__init__.py

```diff
@@ -2,5 +2,7 @@
 # Copyright 2015-2017 Odoo S.A.
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
-from . import models
-from . import report
+from . import (
+    models,
+    report,
+)
```

Comment:
Please, delete this change, because I have some scripts that runs over `__init__` files and I don't want to change them just because you want to use this kind of notation that nobody uses.

Reply:
Although I agree on not changing this to reduce diff, not sure what scripts do you have but those are wrong indeed, this is perfectly valid Python.